CONTRIBUTING.md

```diff
@@ -30,52 +52,96 @@ For more information specific to how a quarto site works, and for examples of wh
+pre-existing anchor link, consider how that will be redirected for users
+who may have bookmarked it
 
 #### Make changes in the UI
 
-Click **Edit this page** at the bottom of the right hand table of contents of any page to make small changes such as a typo, sentence fix, or a broken link. This takes you to the `.Qmd` file where you can make your changes and [create a pull request](#pull-request) for a review.
+Click **Edit this page** at the bottom of the right hand table of
+contents of any page to make small changes such as a typo, sentence fix,
+or a broken link. This takes you to the `.Qmd` file where you can make
+your changes and [create a pull request](#pull-request) for a review.
 
 #### Make changes locally
 
-1. Clone or fork the repository.
+1.  Clone or fork the repository.
 
-2. Open the repository in your editor of choice, e.g. R Studio.
+2.  Open the repository in your editor of choice, e.g. R Studio.
 
-3. Create a working branch and start with your changes!
+3.  Create a working branch and start with your changes!
 
-4. Commit and push the changes to your working branch.
+4.  Commit and push the changes to your working branch.
 
-### Pull Request
+### Pull Request {#pull-request}
 
 All pull requests should be made against the main branch.
 
-When you're finished with the changes, create a pull request, also known as a PR.
-- Don't forget to [link PR to issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) if you are solving one.
-
-Once you submit your PR, a repository admin will review your proposal. We may ask questions or request additional information.
-- We may ask for changes to be made before a PR can be merged, either using [suggested changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request) or pull request comments. You can apply suggested changes directly through the UI. You can make any other changes and then commit them to your branch.
-- As you update your PR and apply changes, mark each conversation as [resolved](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#resolving-conversations).
-- If you run into any merge issues, checkout this [Git tutorial](https://github.com/skills/resolve-merge-conflicts) to help you resolve merge conflicts and other issues.
-
-## Support 
-
-If you need any assistance at all, please contact [statistics.development@education.gov.uk](mailto:statistics.development@education.gov.uk).
\ No newline at end of file
+When you're finished with the changes, create a pull request, also known
+as a PR. - Don't forget to [link PR to
+issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
+if you are solving one.
+
+Once you submit your PR, a repository admin will review your proposal.
+We may ask questions or request additional information. - We may ask for
```

Comment:
Think the bullet points have been lost here? 

Preview of this:
![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/8b43e3ed-dbb6-4165-a846-24d50eaf1587)

What it looks like on main (https://github.com/dfe-analytical-services/analysts-guide/blob/main/CONTRIBUTING.md)

![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/1dd21240-38a7-4b1c-b9b9-6293efa1423f)